content/influxdb/v2.0/reference/cli/influx/task/delete.md

```diff
@@ -16,17 +16,18 @@ influx task delete [flags]
 ```
 
 ## Flags
-| Flag |                   | Description                                                           | Input type  | {{< cli/mapped >}}    |
-|:---- |:---               |:-----------                                                           |:----------: |:------------------    |
-| `-c` | `--active-config` | CLI configuration to use for command                                  | string      |                       |
-|      | `--configs-path`  | Path to `influx` CLI configurations (default `~/.influxdbv2/configs`) | string      |`INFLUX_CONFIGS_PATH`  |
-| `-h` | `--help`          | Help for the `delete` command                                         |             |                       |
-|      | `--hide-headers`  | Hide table headers (default `false`)                                  |             | `INFLUX_HIDE_HEADERS` |
-|      | `--host`          | HTTP address of InfluxDB (default `http://localhost:8086`)            | string      | `INFLUX_HOST`         |
-| `-i` | `--id`            | ({{< req >}}) Task ID                                                 | string      |                       |
-|      | `--json`          | Output data as JSON (default `false`)                                 |             | `INFLUX_OUTPUT_JSON`  |
-|      | `--skip-verify`   | Skip TLS certificate verification                                     |             |                       |
-| `-t` | `--token`         | Authentication token                                                  | string      | `INFLUX_TOKEN`        |
+| Flag |                   | Description                                                           | Input type | {{< cli/mapped >}}    |
+| :--- | :---------------- | :-------------------------------------------------------------------- | :--------: | :-------------------- |
+| `-c` | `--active-config` | CLI configuration to use for command                                  |   string   |                       |
+|      | `--configs-path`  | Path to `influx` CLI configurations (default `~/.influxdbv2/configs`) |   string   | `INFLUX_CONFIGS_PATH` |
+| `-h` | `--help`          | Help for the `delete` command                                         |            |                       |
+|      | `--hide-headers`  | Hide table headers (default `false`)                                  |            | `INFLUX_HIDE_HEADERS` |
+|      | `--host`          | HTTP address of InfluxDB (default `http://localhost:8086`)            |   string   | `INFLUX_HOST`         |
+|      | `--http-debug`    | Inspect communication with InfluxDB servers.                          |   string   |                       |
+| `-i` | `--id`            | ({{< req >}}) Task ID                                                 |   string   |                       |
```

Comment:
what's with the `({{< req >}})` in this line? i don't see it elsewhere.

Reply:
shortcode to identify required parameters
https://github.com/influxdata/docs-v2/blob/master/CONTRIBUTING.md#required-elements